internal/cloudsql/refresh.go

```diff
@@ -47,9 +47,12 @@ const (
 // metadata contains information about a Cloud SQL instance needed to create
 // connections.
 type metadata struct {
-	ipAddrs      map[string]string
-	serverCaCert *x509.Certificate
-	version      string
+	ipAddrs         map[string]string
+	serverCaCert    *x509.Certificate
```

Comment:
Same as above, use a single field here instead of two fields: `serverCaCert` and `serverCaCertPem`

Reply:
SG. I used the `serverCACert` now.